build.sbt

```diff
@@ -114,18 +121,17 @@ ThisBuild / credentials += Credentials(
     "GHPR_TOKEN required to fetch or publish from/to GitHub Package Registry"))
 
 // Sonatype settings
-qbeastSparkMaven / publishMavenStyle := true
-qbeastSparkMaven / sonatypeProfileName := "io.qbeast"
-qbeastSparkMaven / sonatypeProjectHosting := Some(
+ThisBuild / publishMavenStyle := true
+ThisBuild / sonatypeProfileName := "io.qbeast"
+ThisBuild / sonatypeProjectHosting := Some(
```

Comment:
Same as this. It only affects the `qbeastSparkMaven` project (sonatype configs).

Reply:
You're right, sorry! I got confused seeing different commits 🤭